prepare release for resourcemanager (#27868)
Azure/azure-sdk-for-net: sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md

# Release History

## 1.0.0-beta.9 (2022-03-31)

### Features Added

- New struct `ArmEnvironment`.

> ### Breaking Changes

Reviewer: It looks like that we missed the change `Objetc` to `BinaryData`

- Now all the resource classes will have a `Resource` suffix (if it previously does not have one).

> - Rename some model to a more comprehensive name.

Reviewer 2: ```suggestion
- Rename some models to more comprehensive names.
```

> - Rename some models to more comprehensive names.

Reviewer: Maybe `Renamed` is better? Because the previous changelogs all used the past tense.

Author: I jsut checked the changelog of azure.core. They are using past tense.
I will change the tense here

> - Class `ManagementGroupResource` (previously `ManagementGroup`), `ManagementGroupCollection` and `ManagementGroupData` move from `Azure.ResourceManager.Management` namespace to `Azure.ResourceManager.ManagementGroups`.

Reviewer 2: ```suggestion
- Class `ManagementGroupResource` (previously `ManagementGroup`), `ManagementGroupCollection` and `ManagementGroupData` are moved from `Azure.ResourceManager.Management` namespace to `Azure.ResourceManager.ManagementGroups`.
```

> - Class `ArmResource` and `ArmCollection` move from `Azure.ResourceManager.Core` to `Azure.ResourceManager`.

Reviewer 2: ```suggestion
- Move class `ArmResource` and `ArmCollection` from `Azure.ResourceManager.Core` to `Azure.ResourceManager`.
```

> - Namespace `Azure.ResourceManager.Core` and `Azure.ResourceManager.Management` are removed.

Reviewer 2: ```suggestion
- Remove namespace `Azure.ResourceManager.Core` and `Azure.ResourceManager.Management`.
```

> - Class `ErrorDetail` and `ErrorAdditionalInfo` are removed.

Reviewer 2: ```suggestion
- Remove class `ErrorDetail` and `ErrorAdditionalInfo`.
```
